FeedAPIChallenge/Feed API/FeedImagesMapper.swift

```diff
@@ -0,0 +1,42 @@
+//
+//  FeedImagesMapper.swift
+//  FeedAPIChallenge
+//
+//  Created by Pavan Powani on 03/08/21.
+//  Copyright © 2021 Essential Developer Ltd. All rights reserved.
+//
+
+import Foundation
+
+class FeedImagesMapper {
+	struct ImagesRoot: Decodable {
+		private var items: [Item]
+
+		var feedImages: [FeedImage] {
+			return items.map({ $0.feedImage })
+		}
+	}
+
+	fileprivate struct Item: Decodable {
```

Comment:
`Item` is not being accessed in any other part of this file, so it would be preferable to mark it as `private`.
```suggestion
	private struct Item: Decodable {
```